install/Docker/dockerfiles/scripts/consul-entrypoint.sh

```diff
@@ -13,7 +13,6 @@ set -e
 
 # If deployment type is HA mode, need to add the other consul instances
 # to form the quorum
-
```

Comment:
whats the change ?

Reply:
Expecting IS_CONSUL_CLUSTER to have true or false set, by including ",," value set is converted to lowercase and is used for making case insensitive match.

Reply:
And also deleted am empty line, for github to retrigger actions, as it was hung for long time and re-run wasn't available.